src/c/ckb_smt.c

```diff
@@ -15,6 +15,14 @@ smt_state_t* smt_state_new(uint32_t capacity) {
   return state;
 }
 
+void smt_state_del(smt_state_t* state) {
```

Comment:
I don't think we have used "malloc" "free" in SMT. If it's not used, just remove it.